### DISCUSSION TITLE
Anyway to stop Postal from re-trying bounces?

### DISCUSSION THREAD

**Discussion by nanohits** [postalserver/postal#2145] [Help with using Postal]
Hi,
Is there any way for Postal to stop retrying to send bounced emails again till it hard fails?
I have looked in the settings but could not find.

**Comment by willpower232** [comment]
Postal is retrying because it thinks the bounce can be overcome by retrying, I believe the amount of times is controlled in the config file https://github.com/postalserver/postal/blob/6bace2c9053e8bdfc2689c3c92a9fc066d124401/config/postal.defaults.yml#L14

**Reply by ghost** [reply]
Is there an internal way to make it stop? For example, to "clear the queue", would I need to remove messages from a table or set a status?
